lib/graphql/formatter/snake_case.ex

```diff
@@ -0,0 +1,41 @@
+defmodule BridgeEx.Graphql.Formatter.SnakeCase do
+  @moduledoc """
+  Formatter to format map keys to snake_case.
+  """
+
+  @behaviour BridgeEx.Graphql.Formatter.Adapter
+
+  alias BridgeEx.Graphql.LanguageConventions
+
+  @spec format(map()) :: map()
+  def format(variable) when is_map(variable) do
+    variable
+    |> Enum.map(fn
+      {key, value} -> {format_key(key), format(value)}
+    end)
+    |> Map.new()
+  end
```

Comment:
Forse anche qui si può usare `Enumerable.impl_for(variable)` come il formatter CamelCase?

Reply:
Essendo che dalla risposta non tornano mai dei tipi complessi ma solo dei tipi primitivi non si incappa mai nel caso in cui una mappa non sia enumerabile.